src/Commands/Builder.php

```diff
@@ -144,6 +144,35 @@ public function where($property, $value = null, $operator = '=', $conjunction =
     }
 
     /**
+     * Set the type of the target in the current Command Bag
+     * @param $type
+     * @return $this
+     */
+    public function type($type)
```

Comment:
I'm not against keeping `Builder::type()` but this refers to the deprecated `target`.
I'm also not against ditching `type()` altogether